client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -489,12 +490,16 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow } ) => {
 						_selectedDesign,
 						null,
 						PLACEHOLDER_SITE_ID
-					).then( () => reduxDispatch( requestActiveTheme( site?.ID || -1 ) ) );
+					).then( ( theme: ActiveTheme ) =>
+						reduxDispatch( setActiveTheme( site?.ID || -1, theme ) )
```

Comment:
Maybe for readability would be nice to have a variable explaining the -1.

Reply:
I honestly don't know, and it's not my code.

Reply:
Thanks for the feedback. We'll take care of that. cc: @arthur791004